model/cicd/spans.yaml

```
@@ -0,0 +1,45 @@
+groups:
+  - id: span.cicd.pipeline.run.server
+    type: span
+    span_kind: server
+    brief: This span describes a CICD pipeline run.
+    note: |
+      For all pipeline runs, a span with kind `SERVER` SHOULD be created corresponding to the execution of the pipeline run.
+
+      **Span name** MUST follow the overall [guidelines for span names](https://github.com/open-telemetry/opentelemetry-specification/tree/v1.43.0/specification/trace/api.md#span).
+
+      The span name SHOULD be `{action} {pipeline}` if there is a (low-cardinality) pipeline name available.
+      If the pipeline name is not available or is likely to have high cardinality, then the span name SHOULD be `{action}`.
+
+      The `{action}` SHOULD be the [`cicd.pipeline.action`](/docs/attributes-registry/cicd.md#cicd-pipeline-action).
+
+      The `{pipeline}` SHOULD be the [`cicd.pipeline.name`](/docs/attributes-registry/cicd.md#cicd-pipeline-name).
```

moving https://github.com/open-telemetry/semantic-conventions/pull/2013#discussion_r2067121403 discussion here, why not

```suggestion
      The span name SHOULD be `{cicd.pipeline.action} {cicd.pipeline.name}` if there is a (low-cardinality) pipeline name available.
      If the pipeline name is not available or is likely to have high cardinality, then the span name SHOULD be `{cicd.pipeline.action}`.
```

in HTTP there are multiple options for each component, which is not the case here.

Check out DB : https://github.com/open-telemetry/semantic-conventions/blob/main/docs/database/database-spans.md#name

---

Having tried this in ac67d6bf457a90fa80228b3aeefee9c8efbf40d0 I noticed that the markdown did not display the link to the attribute registry well.
I thus reverted it in 61f48f92df41f2c94e96bb9269d1b5f5a32ae99e since I preferred to have the link to the attribute registry.